docs/library/pyb.Pin.rst

```diff
@@ -107,7 +101,8 @@ Methods
         - ``Pin.IN`` - configure the pin for input;
         - ``Pin.OUT_PP`` - configure the pin for output, with push-pull control;
         - ``Pin.OUT_OD`` - configure the pin for output, with open-drain control;
-        - ``Pin.AF_PP`` - configure the pin for alternate function, pull-pull;
+        - ``Pin.ALT`` - configure the pin for alternate function, input or output;
+        - ``Pin.AF_PP`` - configure the pin for alternate function, push-pull;
```

Comment:
ALT and AF_PP are equivalent, they are aliases of each other.  So probably best to describe them both as "input or output".

There's also the newer/preferred ALT_OPEN_DRAIN which aliases AF_OD.

Reply:
Well, it's probably worth just getting rid of all the legacy names in the docs here.  The newer/preferred are IN, OUT, OPEN_DRAIN, ALT, ALT_OPEN_DRAIN, ANALOG.

Reply:
This change is rather involved and can be done later.